src/cmp_aus.js

```diff
@@ -161,30 +220,41 @@ const checkPage = async (pageType, url) => {
 	await clearCookies(page);
 
 	// Now we can run our tests.
-	log(`Test 1 start: Adverts load and the CMP is displayed on initial load`);
+	log(
+		`[TEST 1] start: CMP loads and the ads are NOT displayed on initial load`,
```

Comment:
In Australia, the ads are supposed to load before the CMP has been interacted with